test/functional/apps.js

```diff
@@ -0,0 +1,20 @@
+define([
+  'intern!object',
```

Comment:
Do you know if intern supports the common.js style AMD module constructors? It's just using require.js under the covers, right?

If so we should update this to follow that established pattern throughout `src`.